From 5d5c83c89be0cff29dde2267fa168e6f7146be8c Mon Sep 17 00:00:00 2001
From: Robert Yang <liezhi.yang@windriver.com>
Date: Mon, 4 May 2015 22:31:48 -0700
Subject: [PATCH] base/gendev.c: fix for -Werror=return-type

Fixed:
base/gendev.c:80:1: error: return type defaults to 'int' [-Werror=return-type]

Upstream-Status: Pending

Signed-off-by: Robert Yang <liezhi.yang@windriver.com>
---
 base/gendev.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/base/gendev.c b/base/gendev.c
index 68a745f..8058b5a 100644
--- a/base/gendev.c
+++ b/base/gendev.c
@@ -77,6 +77,7 @@ static const char *indent_item = "";
 /* Forward definitions */
 void add_entry(config *, const char *, const char *, bool);
 
+int
 main(int argc, char *argv[])
 {
     config conf;
-- 
1.7.9.5

